### ISSUE TITLE
Edited Pals lose abilities.

### ISSUE THREAD

**Issue by thesimpleway1** [EternalWraith/PalEdit#68]
Edited a Frostallion for its HP. After loading the file I removed his Crystal wing ability from its active abilities by accident and tried to put it back.

It disappeared. It hasn't the ability anymore. 

It's an ability you can not learn through a fruit, you can reverse that.

**Comment by thesimpleway1** [comment]
Update for v.0.5.3:

Still deletes abilities, even from pals that are not in any way connected to the ones I edited.

**Comment by Anson0627** [comment]
I am the same problem here

**Comment by Snowstar38** [comment]
I made a PR that will hopefully fix this, it seems to work for me.

**Comment by Anson0627** [comment]
Is there any solution now?

**Comment by thesimpleway1** [comment]
> Is there any solution now?

No. 0.5.5 is still doing that. I would recommend waiting until there is an update that specifically address this issue.

**Comment by Snowstar38** [comment]
If you have Python installed on your PC you can go to my PalEdit repo that has the proposed fix I made in it. Download the zip and double click the .bat file to run.

**Comment by lord-emperor** [comment]
Can confirm all edited pals lose all their abilities. In-game solution is to breed them but this means leveling the offspring up from 1 again.

**Comment by krulci** [comment]
> If you have Python installed on your PC you can go to my PalEdit repo that has the proposed fix I made in it. Download the zip and double click the .bat file to run.

Can you please add a feature to unlock all learnable skills?

**Comment by krulci** [comment]
> If you have Python installed on your PC you can go to my PalEdit repo that has the proposed fix I made in it. Download the zip and double click the .bat file to run.

blazamut still has issue. It seems like it will learn the wrong skill. Choosable skills are not empty. It has three skills; however, the skill is wrong.
I've assigned Fire Ball/Ignis Rage/Ignis Blast, but it came out with something else
